src/test/java/core/basesyntax/OperatorsAndStrategyTest.java

```diff
@@ -0,0 +1,145 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operators.BalanceOperationHandler;
+import core.basesyntax.operators.OperationHandler;
+import core.basesyntax.operators.PurchaseOperationHandler;
+import core.basesyntax.operators.ReturnOperationHandler;
+import core.basesyntax.operators.SupplyOperationHandler;
+import core.basesyntax.service.OperatorStrategy;
+import core.basesyntax.service.impl.OperatorStrategyImpl;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class OperatorsAndStrategyTest {

```

Comment:
All your handlers should be test isolated.  
also to test strategy follow this common mistake solution 
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/32339836/705c3468-69a2-45a6-a443-b28d8402dcdf)